lib/internal/checkUrl.js

```diff
@@ -24,6 +24,23 @@ function checkUrl(link, baseUrl, options, callback)
 		callHandler.async(callback, link);
 		return;
 	}
+
+	if (cache)
```

Comment:
`!== undefined`

It's an internal function, so we probably don't need `typeof`